src/webui/www/private/scripts/client.js

```diff
@@ -578,8 +622,19 @@ window.addEvent('load', function() {
         }
         trackerFilterList.appendChild(createLink(TRACKERS_TRACKERLESS, 'QBT_TR(Trackerless (%1))QBT_TR[CONTEXT=TrackerFiltersList]', trackerlessTorrentsCount));
 
-        for (const [hash, tracker] of trackerList)
-            trackerFilterList.appendChild(createLink(hash, tracker.url + ' (%1)', tracker.torrents.length));
+        // Sort trackers by hostname
+        const sortedMap = new Map([...trackerList.entries()].sort((a, b) => {
+            const xHost = getHost(a[1].url.toLowerCase());
+            const yHost = getHost(b[1].url.toLowerCase());
+            if (xHost < yHost)
+                return -1;
+            if (xHost > yHost)
+                return 1;
+            return 0;
```

Comment:
```suggestion
            return xHost > yHost;
```
ps. and personally I would name them `left` and `right` but this isn't very important.

Reply:
resoved https://github.com/qbittorrent/qBittorrent/pull/18190/commits/0fc08d1415410d436e42bfde3cc10883024128b6